public/sections/todo/todo.html

```diff
@@ -4,19 +4,23 @@
         <input type="text" placeholder="Search" ng-model="findTodo">
     </li>
     <li class="collection-item" ng-repeat="todo in ctrl.todos | filter : findTodo">
-        <a><i class="material-icons waves-circle tiny padding-circle" style="background-color : {{todo.color}}; ">circle</i></a>
+        <a href="" style="background-color : {{todo.color}}; " class="material-icons tiny btn-floating"></a>
         <strong> {{todo.name}}</strong>
-        <a href="" class="secondary-content padding-edit-button" ><i class="material-icons">edit</i></a>
+        <a href="" class="secondary-content padding-edit-button"><i class="material-icons">edit</i></a>
     </li>
-
-
 </ul>
 
 <div class="fixed-action-btn">
     <a class="btn-floating btn-large blue-grey">
```

Comment:
We should probably avoid null `href` attributes, something like `href="javascript:void(0);` should work better

Or we could use an actual `<button>` as well.